### PR TITLE
Fix validation of payloads containing floats

### PR DESCRIPTION
**Pull request by OrangeTux**
The validation of payloads using jsonschemas could fail when the payload
contained a float. This problem is described in this issue:
https://github.com/Julian/jsonschema/issues/247

This commit implements a work around for this issue by changing the
float parser for certain payloads from `float()` to `decimal.Decimal()`.

Fixes: #43

### CODE REVIEW

File: ocpp/messages.py [hunk 1]
```diff
@@ -32,7 +33,7 @@ def unpack(msg):
     Unpacks a message into either a Call, CallError or CallResult.
     """
     try:
-        msg = json.loads(msg)
+        msg = json.loads(msg, parse_float=decimal.Decimal)
```
Comment (proelke) [inline ocpp/messages.py]: Is it ok to always parse floats here but only schemas in a few instances?

Comment (OrangeTux) [reply ocpp/messages.py]: Good catch, I need to verify that.

Comment (proelke) [reply ocpp/messages.py]: I started working on this a bit yesterday, and I got stuck on that part of porting the code over. If it's an issue I have an idea.

Comment (OrangeTux) [reply ocpp/messages.py]: I've removed the argument `parse_float=decimal.Decimal`. It is not needed in this place.

If a message requires special validation regarding the floats the message is reparsed again using `parse_float=decimal.Decimal`. See line 138 of this file.

**Pull request merged**